app/models/manageiq/providers/base_manager/metrics_capture.rb

```diff
@@ -199,23 +222,27 @@ def perf_capture_queue_target(target, interval_name, start_time:, end_time:, tas
       :zone        => my_zone,
       :state       => ['ready', 'dequeue'],
     }
-    queue_item[:args] = [start_time, end_time] if start_time
+    queue_item[:args] = [start_time, end_time, target_ids, rollup] if start_time || target_ids.present? || rollup
+
+    # GOAL only use put/put_unless_exists and drop put_or_update
```

Comment:
Is this comment needed?  Seems you met your goal since the method used it put_unless_exists.

Reply:
probably not. Wanted to make it clear that we want to delete everything after this message.

I'll remove